docs/docs/dcap.md

```diff
@@ -1,96 +1,73 @@
+want to run your code on bare-metal machines and don't want to rely on cloud providers.
 
-Microsoft has written a DCAP client plugin that works with its Azure Confidential Compute virtual machines. In fact, it also works outside of Azure for single CPU systems, but this may not always be the case.
+To use Intel's DCAP plugin, you need to install the DCAP client package. For example, in Ubuntu, you can 
+use the following command:
 
-Conclave's bundled Azure client plugin will *only* be used if no other plugin has been found on the system.
-The runtime will use the first `.so` it encounters in the search order below:
-```
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
-/usr/lib/libdcap_quoteprov.so.1
-/usr/lib/libdcap_quoteprov.so
-```
-Should you decide to use a bundled version (recommended), ensure the files listed above don't exist (delete or rename
-them if necessary).
+`sudo apt-get install libsgx-dcap-default-qpl`
 
-!!!Important
+You also need a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification) to use 
+Intel's DCAP plugin. You must also set up a Provisioning Certificate Caching Service (PCCS). Intel provides an 
+example and some instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). 
+If you like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs),
+then you might need to provide data in the correct format for Conclave. Please [let us know](mailto:conclave@r3.com) if 
+you need to set this up.
 
-    The bundled DCAP Azure client plugin works with all current generation Azure confidential VMs, as well as existing
-    CPUs. However, if Azure extends their offerings to include new platforms, or if you use a new generation system, 
-    then the bundled Azure client plugin may throw an error when attempting to load your enclave. In this case,
-    [follow the instructions below](#azure-client-plugin) to build/install an updated version of the Azure client 
-    plugin that supports the platform.
+### Azure DCAP Client
 
-You may want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
-```sh
-export AZDCAP_DEBUG_LOG_LEVEL=FATAL
-```
+Microsoft provides an Azure DCAP client, which you can use on Azure virtual machines. Azure's DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) 
+does not require any subscription to Intel services. 
 
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
+To use an Azure DCAP client, you need to install Azure's DCAP client package. For example, in Ubuntu, you can use 
+the following command:
 
-#### Azure client plugin
+`sudo apt-get install az-dcap-client`
```

Comment:
I don't believe this will work just on its own. The user will need to first install the Microsoft packages list, or otherwise it won't recognise `az-dcap-client`.

```
wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
echo 'deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod focal main' | sudo tee /etc/apt/sources.list.d/msprod.list
```